src/test/testUtil.ts

```diff
@@ -243,25 +243,27 @@ export async function assertTextEditorContains(contents: string): Promise<void |
 }
 
 /**
- * Executes the close all editors command and waits for all visible editors to disappear
+ * Executes the "openEditors.closeAll" command and asserts that all visible
+ * editors were closed after waiting.
  */
-export async function closeAllEditors(): Promise<unknown> {
-    if ((await vscode.commands.getCommands()).includes('openEditors.closeAll')) {
+export async function closeAllEditors(): Promise<void> {
+    const hasCloseAll = (await vscode.commands.getCommands()).includes('openEditors.closeAll')
+    // Derived by inspecting 'Keyboard Shortcuts' via command `>Preferences: Open Keyboard Shortcuts`
+    // `workbench.action.closeAllEditors` is unreliable and should not be used if possible
+    const closeAllCmd = hasCloseAll ? 'openEditors.closeAll' : 'workbench.action.closeAllEditors'
+    if (hasCloseAll) {
         if (isMinimumVersion() && !isReleaseVersion()) {
-            throw Error('openEditors.closeAll is available in min version, remove me!')
+            throw Error(
+                '"openEditors.closeAll" is available in min version, remove use of "workbench.action.closeAllEditors"!'
+            )
```

Comment:
Could move to `techdebt.test.ts` though I believe the new command will be available once we bump minver to whatever Cloud9 is on